android/sentry.properties

```diff
@@ -0,0 +1,4 @@
+defaults.url=https://sentry.io/
+defaults.org=nebra
+defaults.project=nebra-hotspot-mobile-app
+auth.token=87ff67bdfc074a25b471f8b358d889a26612c2f9228b4c598af7b8cd90431e86
```

Comment:
Why can't these variables be pulled from `.env`? Should this file really be in source control?